### ISSUE TITLE
Replace upstream binaries with packages in the kindbase image

### ISSUE THREAD

**Issue by afbjorklund** [kubernetes/minikube#8767]
Currently kindbase installs containerd, crictl and cni-plugins from upstream binaries.

```Dockerfile
# Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
# The repository contains latest stable releases and nightlies built for multiple architectures
ARG CONTAINERD_VERSION="v1.3.3-14-g449e9269"
# Configure CNI binaries from upstream
ARG CNI_VERSION="v0.8.5"
# Configure crictl binary from upstream
ARG CRICTL_VERSION="v1.17.0"
```

```sh
# Then we install containerd from our nightly build infrastructure, as this
# build for multiple architectures and allows us to upgrade to patched releases
# more quickly.
#
# Next we download and extract crictl and CNI plugin binaries from upstream.
```

We should use the the deb packages instead, so that they work as dependencies...

```
NAME                                VERSION                            SIZE
-containerd                         1.3.3-0ubuntu1~19.10.2             128.1M
-containernetworking-plugins        0.8.6~1                            49.8M
-cri-tools                          1.17.0~3                           22M
```

https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable

Right now we have both.

i.e. `/usr/local` **and** `/usr`.

**Comment by afbjorklund** [comment]
See https://github.com/kubernetes/minikube/pull/8251#issuecomment-643815675

**Comment by afbjorklund** [comment]
Same problem in #7884

**Comment by afbjorklund** [comment]
See https://github.com/kubernetes/minikube/pull/8251#issuecomment-643815675

**Comment by afbjorklund** [comment]
Included in PR #9330
